sources/CMLTYPES

```diff
@@ -394,10 +398,13 @@
                 (LIST 'ARRAY (ARRAY-ELEMENT-TYPE ARRAY)
                       (ARRAY-DIMENSIONS ARRAY))))))
 
-(DEFUN SYMBOL-TYPE (SYMBOL)
-   (IF (KEYWORDP SYMBOL)
-       'KEYWORD
-       'SYMBOL))
+(DEFUN SYMBOL-TYPE (SYMBOL)                           (IL:* IL:\; "Edited  4-Jun-2024 23:23 by mth")
+   (COND
+      ((NULL SYMBOL)
+       'NULL)
+      ((KEYWORDP SYMBOL)
+       'KEYWORD)
+      (T 'SYMBOL)))
```

Comment:
This appears to be correct and match the expectations in CLtL2 2.15 (Overlap, Inclusion, and Disjointness of Types) on p.39, which states "The type null is a subtype of symbol; the only object of type null is nil" (the first part of this statement was already true) and p.66, which states that `(subtypep (type-of nil) 'null)` must return `T T`.

Reply:
Yes, the changes due to the file being rewritten by `MAKEFILE` (including as from within `CLEANUP`) are considered as _ancillary_ and are acceptable.

Reply:
Since there’s no way on GitHub to control its diff interpretation of “whitespace”, such as the font shifts, and prettyprint layout changes, we use a lisp based set of tools in medley to review changes to lisp source.

Reply:
I'd be interested to know about that tool, it was a lot of scrolling through that diff to find the real change!  Is there a process guide here somewhere?

Reply:
I thought I had seen a note (possibly email to @eblanton ?) 👍🏽 
THe tool is (in lispsuers). in the file GITFNS.

(FILESLOAD GITFNS). should load it in. The Exec command "prc (either case and in any package) will bring up a pull-request menu.